R2API/ContentManagement/R2APIContentManager.cs

```diff
@@ -45,12 +48,14 @@ public struct ManagedReadOnlyContentPack {
         /// <summary>
         /// The ReadOnlyContentPack
         /// </summary>
-        public ReadOnlyContentPack contentPack;
+        public ReadOnlyContentPack ContentPack { get; }
```

Comment:
public API change, please confirm if this was intended

Reply:
Intended change.

Even if it wasn't intended, the Content management system hasn't been released to the public yet, which imo means we shouldn't worry too much about that stuff